tests/test_stationdata.py

```diff
@@ -1,15 +1,23 @@
+from contextlib import nullcontext as does_not_raise_exception
+
 import numpy as np
-import pytest
 import pandas as pd
-from xarray import DataArray
```

Comment:
This looks wrong since it's used here:
https://github.com/metno/pyaerocom/blob/064d539f4d7dbc9ff390b89aeef1ad7b7031e6d7/tests/test_stationdata.py#L261

Reply:
I `isort`ed the file. The import is still there, after `from matplotlib.axes import Axes`.

Reply:
Then I have misread the file, sorry for that.